doc/source/user/index.rst

```diff
@@ -19,3 +19,4 @@ classes contained in the package, see the :ref:`reference`.
    numpy-for-matlab-users
    building
    c-info
+   absolute_beginners
```

Comment:
Can you move this up? I think the best place is right under `quickstart`.

Reply:
Done!